HTTPServerProject.Tests/HTTPServerResponseTests/TestParameters.cs

```diff
@@ -0,0 +1,95 @@
+using HTTPServerResponse.Parameters;
+
+namespace HTTPServerResponseTests.Parameters;
+
+public class TestPathParameters : IPathParameters
+{
+    public Dictionary<string, Dictionary<string, Dictionary<string, object>>> pathDict = new Dictionary<string, Dictionary<string, Dictionary<string, object>>>();
```

Comment:
You can use `using` to make this type way shorter, for example `using MyAlias = System.Collections.Generic.List<System.Tuple<int, string, int>>;`

Reply:
I couldn't get this to work for this constructor variable.